worker/worker.py

```diff
@@ -352,7 +389,17 @@ def main():
     print("Worker version {} connecting to {}".format(WORKER_VERSION, remote))
 
     try:
-        cpu_count = min(int(options.concurrency), multiprocessing.cpu_count() - 1)
+        if options.use_all_cores == "True":
+            cpu_count = multiprocessing.cpu_count()
+        else:
+            cpu_count = int(options.concurrency)
+            if cpu_count > multiprocessing.cpu_count() - 1:
+                sys.stderr.write("You cannot have concurrency {} but at most {}\n", options.concurrency, multiprocessing.cpu_count() - 1)
```

Comment:
maybe refer to the other option ?
"You cannot have concurrency {} but at most {}, override this check with the option --use_all_cores\n"

Reply:
Done, thanks.